xbmc/windowing/wayland/WinSystemWayland.cpp

```diff
@@ -379,7 +379,7 @@ bool CWinSystemWayland::CreateNewWindow(const std::string& name,
   // There are two stages to the event handling:
   // * Initialization (which ends here): Everything runs synchronously and init
   //   code that needs events processed must call roundtrip().
-  //   This is done for simplicity because it is a lot easier than to make
+  //   This is done for simplicitly because it is a lot easier than to make
```

Comment:
This change seems wrong

Reply:
Thanks for the catch! I swear I removed that line off the lintfile. Fixed!